src/target/stm32g0.c

```diff
@@ -206,11 +215,30 @@ bool stm32g0_probe(target_s *t)
 	size_t flash_size = 0U;
 
 	switch (t->part_id) {
-	case STM32G03_4:
-		/* SRAM 8kiB, Flash up to 64kiB */
-		ram_size = RAM_SIZE_G03_4;
-		flash_size = FLASH_SIZE_MAX_G03_4;
-		t->driver = "STM32G03/4";
+	case STM32G03_4:;
+		uint32_t dev_id = target_mem_read32(t, DBG_IDCODE) & 0xfffU;
```

Comment:
Please make `dev_id` `const` (`const uint32_t dev_id`)

Reply:
Thanks! Done - and re-pushed.